tests/unit/lms/views/lti/deep_linking_test.py

```diff
@@ -211,9 +270,13 @@ def test_it_for_v11(
                 {"group_set": "1", "title": "title"},
                 {"group_set": "1", "title": "title"},
             ),
+            (
+                {"auto_grading_config": {"key": "value"}},
+                {"auto_grading_config": '{"key": "value"}'},
+            ),
         ],
     )
-    def test_get_assignment_configuration(
+    def test__get_assignment_configuration(
```

Comment:
Fixed this test's name, it testing `_get_assignment_configuration` not `get_assignment_configuration``